Dockerfile

```diff
@@ -29,24 +38,26 @@ RUN apt-get update && \
     apt-get -qy install --fix-missing --no-install-recommends \
         curl \
         && \
-    curl -sL https://deb.nodesource.com/setup_0.10 | bash -
+    curl -sL https://deb.nodesource.com/setup_iojs_2.x | bash -
 
-# install requirements from repos
-# also clean up apt
+#  - package cache updates
+#  - install requirements from repos
+#  - clean up
+#  - install python requirements
+#  - install nodejs requirements
 RUN apt-get update && \
     apt-get -qy upgrade --fix-missing --no-install-recommends && \
     apt-get -qy install --fix-missing --no-install-recommends \
         gcc \
         git \
+        iojs \
         libffi-dev \
         liblzma-dev \
         libmysqlclient-dev \
         libssl-dev \
         libxslt-dev \
         mysql-client \
-        nodejs \
         poppler-utils \
-        subversion \
```

Comment:
Note that if you remove subversion, the following tests would as-if-fail by wrong skipping: (note leading ERROR)

```
======================================================================
ERROR: test_harvest_with_tag_1 (invenio.testsuite.test_utils_vcs_svn.SVNHarvestTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/invenio/testsuite/test_utils_vcs_svn.py", line 43, in setUp
    raise SkipTest("SVN not found. It probably needs installing.")
SkipTest: SVN not found. It probably needs installing.

======================================================================
ERROR: test_harvest_with_tag_2 (invenio.testsuite.test_utils_vcs_svn.SVNHarvestTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/invenio/testsuite/test_utils_vcs_svn.py", line 43, in setUp
    raise SkipTest("SVN not found. It probably needs installing.")
SkipTest: SVN not found. It probably needs installing.

======================================================================
ERROR: test_harvest_without_tag (invenio.testsuite.test_utils_vcs_svn.SVNHarvestTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/invenio/testsuite/test_utils_vcs_svn.py", line 43, in setUp
    raise SkipTest("SVN not found. It probably needs installing.")
SkipTest: SVN not found. It probably needs installing.
```

Could properly decorate them, but this is a non-Docker issue, so can be attacked later.